uint16  status1
uint16  status2
uint64  sensor_ts
uint64  setpoint_ts
float32 encoder_1
float32 velocity_1
float32 current_1_d
float32 current_1_q
float32 current_1_u
float32 current_1_v
float32 current_1_w
float32 voltage_1
float32 voltage_1_u
float32 voltage_1_v
float32 voltage_1_w
float32 temperature_1
float32 encoder_2
float32 velocity_2
float32 current_2_d
float32 current_2_q
float32 current_2_u
float32 current_2_v
float32 current_2_w
float32 voltage_2
float32 voltage_2_u
float32 voltage_2_v
float32 voltage_2_w
float32 temperature_2
float32 encoder_pivot
float32 velocity_pivot
float32 voltage_bus
uint64  imu_ts
float32 accel_x
float32 accel_y
float32 accel_z
float32 gyro_x
float32 gyro_y
float32 gyro_z
float32 temperature_imu
float32 pressure
float32 current_in
  